recipes/phantasm/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# create variables for this build script
+export TEMP_FN=".tmp"
+export BIN_DIR=$PREFIX/bin
+export PHANTASM_DIR=$PREFIX/phantasm
+export PARAM_FN=$PHANTASM_DIR/param.py
+export ACTIVATE_DIR=$PREFIX/etc/conda/activate.d
+export PYVER=$(python -c 'import sys; v=[str(x) for x in sys.version_info[:]]; print(".".join(v[:2]))')
+
+# move phantasm files to the directory
+mkdir -p $PHANTASM_DIR
+cp -rf ./ $PHANTASM_DIR
+
+# define global variables that point at the executables
+echo "BLASTPLUS_DIR:str = '$BIN_DIR'" > $TEMP_FN
+echo "MUSCLE_EXE:str = '$BIN_DIR/muscle'" >> $TEMP_FN
+echo "FASTTREE_EXE:str = '$BIN_DIR/FastTreeMP'" >> $TEMP_FN
+echo "IQTREE_EXE:str = '$BIN_DIR/iqtree'" >> $TEMP_FN
+
+# define global variables that point at phantasm and xenogi directories
+echo "PHANTASM_DIR:str = '$PHANTASM_DIR'" >> $TEMP_FN
+echo "XENOGI_DIR:str = '$PREFIX/lib/python$PYVER/site-packages/xenoGI'" >> $TEMP_FN
+
+# add the last five lines of param.py (LPSN csv files)
+tail -n 5 $PARAM_FN >> $TEMP_FN
+
+# overwrite param.py with the new file
+mv $TEMP_FN $PARAM_FN
+
+# modify PHANTASM_PY variable so it prints the correct usage
+sed -E "s/^(PHANTASM_PY = ).+$/\1'phantasm'/g" $PHANTASM_DIR/phantasm.py > $TEMP_FN
+mv $TEMP_FN $PHANTASM_DIR/phantasm.py
+
+# modify phantasm so it can be invoked directly from the command line
+echo "#! $(which python3)" | sed "s/ //g" > $TEMP_FN # sed is necessary; cannot say `!$`
+echo "" >> $TEMP_FN
+cat $PHANTASM_DIR/phantasm.py >> $TEMP_FN
+mv $TEMP_FN $PHANTASM_DIR/phantasm
+chmod a+x $PHANTASM_DIR/phantasm
+
+# add phantasm to the user's path
```

Comment:
Why can you not use the prefix/bin this would be already on Path and would simplify this build script a lot.

Reply:
I prefer to keep all the source code in its own folder instead of comingling with everything in `bin`. It seems a lot more organized to me and only requires a few extra commands. It also prevents any unexpected collisions. Like if another package has a python file named `main.py`. PHANTASM will break if that file gets clobbered.

Reply:
But because of that you need to change the PATH via an activate script, which is also very ugly and does not work well with containers. Have you checked if the containers are working?

Reply:
the containers are working. why is it ugly? i am new to `conda` recipes.

Reply:
i'd be happy to plop the `phantasm.py` file into `$PREFIX/bin` but idk how to do that and still have it reference all the files it needs to import in the `phantasm` folder.